src/test/java/com/salesforce/mirus/utils/MockTime.java

```diff
@@ -0,0 +1,120 @@
+/*
+ *  Copyright (c) 2018, salesforce.com, inc.
```

Comment:
If this is a just copy from the Kafka repo we allowed to claim copyright on it like this?  Do we really need this class, or can we just use Mockito?

Reply:
Removed